examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -10,8 +10,11 @@
     AutoTokenizer,
     AutoModelForQuestionAnswering,
     AutoModelForTokenClassification,
-    AutoModelForCausalLM
+    AutoModelForCausalLM,
+    AutoModel
```

Comment:
Where this is being used?

Reply:
Thanks @frankfliu yeah AutoModel is not used, AutoModelForCausalLM used for text generation.